contracts/access/roles/CapperRole.sol

```diff
@@ -0,0 +1,35 @@
+pragma solidity ^0.4.24;
+
+import "../Roles.sol";
+
+
+contract CapperRole {
+  using Roles for Roles.Role;
+
+  Roles.Role private cappers;
+
+  constructor() public {
+    cappers.add(msg.sender);
+  }
+
+  modifier onlyCapper() {
+    require(isCapper(msg.sender));
+    _;
+  }
+
+  function isCapper(address _account) public view returns (bool) {
+    return cappers.has(_account);
+  }
+
+  function addCapper(address _account) public onlyCapper {
+    cappers.add(_account);
+  }
+
+  function renounceCapper() public {
+    cappers.remove(msg.sender);
+  }
+
+  function _removeCapper(address _account) internal {
```

Comment:
We've discussed this extensively with @frangio. Our reasoning goes as follows:

An account with a role should always be able to transfer it. If the account is a contract, then the conditions under which a transfer will be executed will be clear from the code. If it's an EOA, then the user already has the liberty to do whatever she wants with it, from complying to transaction requests to giving up the private keys to someone else, effectively transferring control.

So, transferring can always be done. This includes transferring the role to a contract that will forward all calls from a whitelisted set of addresses, which effectively is an `add` to multiple addresses. We decided to formalize this mechanism, and simply remove `transfer` and make `add` public by default. Note that `remove` plays no part here: at no point does having a role give you the power to take it away from someone else, but it does let you act at someone else's behest.

TL;DR: EOAs shouldn't be trusted, since people can do anything they want, if you _really_ need control over this kind of stuff, the role-havers should also be contracts.